src/pages/patientView/sampleManager.tsx

```diff
@@ -181,6 +181,17 @@ class SampleManager {
     getComponentsForSamples() {
         this.samples.map((sample)=>this.getComponentForSample(sample.id));
     }
+
+    getCfDNASampleIconColor(sampleId:string):string {
+        if (["Liquid_Tumor", "Unknown_Tumor"].includes(
+                this.clinicalDataLegacyCleanAndDerived[sampleId]['SAMPLE_TYPE'] )) {
+            return styles.sampleColorMetastasis;
+        } else if (this.clinicalDataLegacyCleanAndDerived[sampleId]['SAMPLE_TYPE'] === "Pleural_Effusion") {
```

Comment:
@khzhu I just looked in our database could you change the check to look for pleural effusion in the following other attributes as well:

```
CATEGORY_SAMPLE
PRIMARY_SITE
PROCEDURE_TYPE
SAMPLE_SITE_SOURCE
SITE_OF_RECURRENCE
TISSUE_SITE
TUMOR_TISSUE_SITE
```
Ideally make it a pretty loose case insensitive match for something like `.*pleural.*effusion.*`

Reply:
agree! will make changes.